### PR TITLE
Added resource exception

### PR DESCRIPTION
**Pull request by WenzDaniel**
_Before you submit this PR: make sure to put all operations-related information in a wiki-note, a PR should be about code and is publicly accessible_

## What does the code in this PR do / what does it improve?
Adds setup which checks if SSL certificate is expired or not.

### CODE REVIEW

**Comment by WenzDaniel** [comment]
Okay so it is the same here. Sometimes it passes and sometimes it fails.

**Comment by WenzDaniel** [comment]
Okay changed it to always skip for now.

**Comment by jmosbacher** [comment]
I guess [this](https://docs.pytest.org/en/latest/example/simple.html#incremental-testing-test-steps) would be the proper way to handle these types of tests where the test logic depends on some resource which may be available and me not at runtime right? but that's maybe a bit overkill for this small quick fix you are proposing.

File: tests/test_scada.py [hunk 1]
```diff
@@ -1,87 +1,105 @@
...
+        Scada interface as self.sc.
+        """
+        if not straxen.utilix_is_configured('scada','scdata_url',):
+            self.skipTest("Cannot test scada since we have no access to xenon secrets.)")
+        
+        try:
+            self.sc = straxen.SCADAInterface(use_progress_bar=False)
+            self.sc.get_new_token()
+        except requests.exceptions.SSLError:
+            self.skipTest("Cannot reach database since HTTPs certifcate expired.")
```
Comment (WenzDaniel) [inline tests/test_scada.py:105]: @jmosbacher does the same as what you are suggesting. The only problem is that not each query triggers the SSLError for some reason. In a few cases you get back a valid token to query parameters. I do not know why, but this I cannot fix on the client side.

Comment (WenzDaniel) [reply tests/test_scada.py:105]: (But in a later query step the server tells you that the certificate has expired.)

Comment (JoranAngevaare) [reply tests/test_scada.py:105]: This is exactly the issue right:
https://github.com/XENONnT/straxen/pull/754#issuecomment-966905978

Comment (WenzDaniel) [reply tests/test_scada.py:105]: Yes, this is why I put a skip flag at the very beginning of the test class so at the moment they are always skipped until the web API is fixed.

**Comment by WenzDaniel** [comment]
@jmosbacher @JoranAngevaare do you guys approve?

**Comment by jmosbacher** [comment]
Im fine with this

**Pull request merged**